### ISSUE TITLE
Cannot dismount "owned" AMS after mounting

### ISSUE THREAD

**Issue by pschord** [psforever/PSF-LoginServer#303]
```
2019-12-15 21:27:01,901  INFO "sessionId=108" WorldSessionActor - DismountVehicleMsg: DismountVehicleMsg(PlanetSideGUID(5113),Normal,false)
2019-12-15 21:27:01,901  WARN "" WorldSessionActor - DismountVehicleMsg: TR awildchord i3-5113 100/100 50/50 attempted to dismount ams, owned by Some(PlanetSideGUID(5113)): (3000/3000)(0/601) (1)'s seat 0, but was not allowed
```

I was allowed to enter but not leave 😬

**Comment by Mazo** [comment]
Likely fixed in #313

**Comment by NickPSF** [comment]
Fix confirmed
